tasks.py

```diff
@@ -62,11 +62,12 @@ def generate_build_files(ctx):
         ]
 
         # Copy all the plugins to the build dir
-        for plugin in plugins_dirs:
-            plugin_dir_build = project_dir_for_build / f"plugin/{plugin.name}"
-            shutil.copytree(src=plugin, dst=plugin_dir_build)
+        for plugin_dir in plugins_dirs:
+            plugin_name, _ = plugin_dir.name.split("-")
```

Comment:
This means the name of the plugin cannot contain hifens. Is that a restriction we're ok with imposing? If not, you can use something like `.rsplit('-', maxsplit=1)` or a regex.